app/lib/server/server.go

```diff
@@ -144,10 +146,17 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
I know you will hate my comment but... I think it would GREAT if users 
+ have the possibility of setting their own value for the fields instead of "ch" and "epfl"
+ or give a path to an already PEM encoded certificate + key

It'd be really nice if the first option at least would be possible. The second one can probably wait.
What do you think ?